docs/hoon/concepts.md

```diff
@@ -105,52 +137,21 @@ which makes your program more readable anyway.  Also, the dumber
 the compiler, the easier it is for a dumb human to understand
 what the compiler is thinking.
 
-### <a name="twig">`twig`</a> (expression)
-
-A [`twig`](../reference) is a Hoon AST node, the result of parsing a
-source expression.  A Hoon source file encodes one twig.
-
-As the noun that the parser produces, a twig is a tagged union of
-the form `{stem bulb}`, where `stem` (the tag) is a `cord` which
-controls the type of `bulb` (the data).
-
-As text, each twig has both regular and irregular syntax forms.
-Regular syntax can be either *tall* or *flat*: multiline or
-single-line.  Irregular syntax is always flat.   Tall twigs can
-contain flat twigs, but not vice versa.
-
-Regular forms always start with a *sigil*, which is either a
-*keyword* or a *rune* at the programmer's choice.  A keyword is
-the stem cord.
+To compile the hoon from the last subsection into Nock, try the 
+following in `:dojo`:
 
-For most stems, tall regular form has no delimiter or terminator,
-eliminating the `)))))` problem common in functional languages.
-Hoon also uses an unusual "backstep" indentation pattern to
-control left-margin drift.
-
-Hoon is mildly whitespace-sensitive.  It knows the difference
-between no whitespace, one-space, and more-space.  But all cases
-of more-space mean the same thing.  Flat syntax uses one-space,
-tall syntax more-space.
-
-As an example, the regular flat keyword syntax `:call(a b)` means
-the same thing as the flat rune syntax `%-(a b)`, the tall
-keyword syntax
-
-```
-:call  a
-b
 ```
-or the tall rune syntax (here on one line, with double spaces):
-```
-%-  a  b
+(mint:ut %noun [%clhp p=[%sand p=%ud q=2] q=[%sand p=%ud q=17]])
 ```
 
-### <a name="span">`span`</a> (type, as range)
+The `%noun` is type information, and the tagged union after that 
```

Comment:
I don't actually know what the type is supposed to represent. It seems worth mentioning. Is it the inferred output type of the nock formula?

Reply:
It's not that, because entering:

`(mint:ut %noun [%clhp p=[%sand p=%ud q=2] q=[%sand p=%ud q=17]])`

...results in:

`[#t/{@ud @ud} q=[%1 p=[2 17]]]`

That's more type info than we gave mint. I don't know what that first type represents.

Reply:
`%noun` in your example is the type of the subject.

Reply:
Thanks Joe!

Reply:
Turns out `%noun` isn't the subject type, but rather it's type info against which the product type is type-checked.